src/js/index.js

```diff
@@ -534,12 +684,18 @@ const initiate = () => {
     /**
      * Add event listeners to edit, save and modal buttons
      */
-    getHtmlElement('#edit')         .addEventListener('click', () => { edit();                                         }, false);
-    getHtmlElement('#save')         .addEventListener('click', () => { save();                                         }, false);
-    getHtmlElement('#lastEdited')   .addEventListener('click', () => { openModal(historySection, populateHistoryHtml); }, false);
-    getHtmlElement('#closeHistory') .addEventListener('click', () => { closeModal(historySection);                     }, false);
-    getHtmlElement('#settings')     .addEventListener('click', () => { openModal(settingsSection, settingsControl);    }, false);
-    getHtmlElement('#closeSettings').addEventListener('click', () => { closeModal(settingsSection);                    }, false);
+    getHtmlElement('#edit')             .addEventListener('click', () => { edit();                                         }, false);
+    getHtmlElement('#save')             .addEventListener('click', () => { save();                                         }, false);
+    getHtmlElement('#lastEdited')       .addEventListener('click', () => { openModal(historySection, populateHistoryHtml); }, false);
+    getHtmlElement('#closeHistory')     .addEventListener('click', () => { closeModal(historySection);                     }, false);
+    getHtmlElement('#settings')         .addEventListener('click', () => { openModal(settingsSection, settingsControl);    }, false);
+    getHtmlElement('#closeSettings')    .addEventListener('click', () => { closeModal(settingsSection);                    }, false);
+    getHtmlElement('#settings-add')     .addEventListener('click', () => { openModal(notelistSection);                     }, false);
+    getHtmlElement('#closeNotesAdd')    .addEventListener('click', () => { closeModal(noteaddSection);                     }, false);
```

Comment:
It would be best if the input could be focussed after the modal has been opened.